python/requirements_ml_docker.txt

```diff
@@ -8,3 +8,11 @@ tblib
 -f https://download.pytorch.org/whl/torch_stable.html
 torch==1.9.0+cu111
 torchvision==0.10.0+cu111
+
+-f https://data.pyg.org/whl/torch-1.9.0+cu111.html
```

Comment:
Do we need to make these changes to `requirements_dl.txt` (line 6 above)?

Reply:
Since there's only a GPU test, I think it should be fine for now

Reply:
Oh but doesn't that make the instruction in line 6 no longer true? Do we actually want these to be in CPU docker as well? Alternative solution: move these above that line.

Reply:
Updated the comment to reflect the changes!